packages/upload-media/src/store/utils/vips.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import { createWorkerFactory } from '@shopify/web-worker';
+
+/**
+ * Internal dependencies
+ */
+import { ImageFile } from '../../image-file';
+import { getFileBasename } from '../../utils';
+import type { ImageSizeCrop, QueueItemId } from '../types';
+
+const createVipsWorker = createWorkerFactory(
+	() => import( /* webpackChunkName: 'vips-worker' */ '@wordpress/vips' )
+);
+const vipsWorker = createVipsWorker();
+
+export async function vipsConvertImageFormat(
```

Comment:
Where are all these vips utils being used? my ctrl + is failing me?

Reply:
Hmm I might have accidentally removed some parts, will check. But the vips integration will warrant its own PR anyway for the glue code.
Reminder, this here is just a small part.

Reply:
In that case, I'd rather leave this out entirely for now and only add the whole vips integration it its own PR.

Reply:
Back at my computer now to double check. The utils _are_ definitely used. `vipsResizeImage` and `vipsCompressImage` are used in some of the actions. Just `vipsConvertImageFormat` isn't used at the moment.